seisbench/data/obs.py

```diff
@@ -0,0 +1,71 @@
+from seisbench.data.base import BenchmarkDataset
+import seisbench.util
+
+class OBS(BenchmarkDataset):
+    """
+    OBS Benchmark Dataset of local events
+
+    Default component order is 'Z12H'. You can easily omit one component like, e.g., hydrophone by explicitly passing
+    parameter 'component_order="Z12"'. This way, the dataset can be input to land station pickers that use only 3
+    components.
+    """
+
+    def __init__(self, **kwargs):
+
+        citation = (
+            "OBS dataset by Bornstein et al."
```

Comment:
Needs to be changed before merging. I'd use the arxiv link and add a TODO to switch to the journal link once submitter.